ui/docs/reference/core/web-server/mozilla-auth0-login-strategy.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Mozilla-Auth0 Login Strategy
+order: 20
+---
+
+# Mozilla-Auth0 Login Strategy
+
+This login strategy is built around [Auth0](https://auth0.com/) and Mozilla's [Person API](https://github.com/mozilla-iam/person-api).
```

Comment:
I think the link to the PersonAPI is pointing to the v1 version of the endpoint. I wonder if we should link to [cis](https://github.com/mozilla-iam/cis) or [cis/docs/PersonAPI.md](https://github.com/mozilla-iam/cis/blob/2287fea51d5a2c4bea88076012178d50e5a385f2/docs/PersonAPI.md) instead.